tests/queries/test_db_returning.py

```diff
@@ -42,6 +42,16 @@ def test_insert_returning_multiple(self):
             ),
             captured_queries[-1]["sql"],
         )
+        self.assertEqual(
+            captured_queries[-1]["sql"]
+            .split("RETURNING ")[1]
+            .count(
+                connection.ops.quote_name(
+                    ReturningModel._meta.get_field("created").column
+                ),
+            ),
+            1,
+        )
```

Comment:
@timgraham if this is an issue for MongoDB we could use `connection.ops.returning_columns([..., ReturningModel._meta.get_field("created")])` instead.